app/controllers/mediated_pages_controller.rb

```diff
@@ -23,7 +23,7 @@ def update_params
   end
 
   def validate_request_type
-    return if current_request.mediateable? || (Settings.features.migration && current_request.pageable? && params[:action] == 'update')
+    return if current_request.mediateable? || (current_request.pageable? && params[:action] == 'update')
```

Comment:
Can't we pull out this second clause?  Wasn't `Settings.features.migration` being nil causing the rest of this to not be needed?

Reply:
I had originally thought not, but tracing through the `blame` farther than I did at first, yes, I do believe the second clause was added to support the migration work. Change forthcoming. Thanks!

Reply:
Done.